cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -1257,10 +1508,10 @@ pub struct LockfileCheckWarning {
 pub mod test_helpers {
     use super::*;
 
-    pub fn fake_package(
+    pub fn fake_catalog_package_lock<P: From<LockedPackageCatalog>>(
```

Comment:
this started as a more or less convenient way to get either an inner `LockedPackageCatalog` or a wrapping `LockedCatalog` through type inference.
But looking back it seems always returning `LockedPackageCatalog` and calling into in a couple more places would be less disruptive.